src/Http/Http.Extensions/test/RequestDelegateGenerator/SharedTypes.cs

```diff
@@ -1014,3 +1015,17 @@ public class TodoChild : Todo
     public string? Child { get; set; }
 }
 #nullable restore
+
+public class TodoWithExplicitIParsable : IParsable<TodoWithExplicitIParsable>
+{
+    public static TodoWithExplicitIParsable Parse(string s, IFormatProvider provider)
+    {
+        return new TodoWithExplicitIParsable();
+    }
+
+    public static bool TryParse(string s, IFormatProvider provider, out TodoWithExplicitIParsable result)
```

Comment:
This isn't explicit

Reply:
Whoops! Thanks for that. Fixed and it works ;)